src/comments.js

```diff
@@ -990,6 +995,12 @@ function printComments(path, print, options, needsSemi) {
       const text = options.originalText;
       if (util.hasNewline(text, util.skipNewline(text, util.locEnd(comment)))) {
         leadingParts.push(hardline);
+      } else if (util.isClosureCompilerTypeCastComment(text, comment)) {
```

Comment:
I'm wondering if this can instead be handled in `fast-path.js`'s `needsParens` function.